templates/pages/reset-password.jsx

```diff
@@ -28,32 +29,47 @@ var ResetPassword = React.createClass({
   render: function() {
     var emailText = "We've emailed you instructions for creating a new password.";
     var linkQuery = {};
+    var content;
+
     linkQuery.client_id = this.state.queryObj.client_id;
     linkQuery.state = this.state.queryObj.state;
     linkQuery.scopes = this.state.queryObj.scopes;
     linkQuery.response_type = this.state.queryObj.response_type;
-    return (
-      <div>
-        <Header origin="Reset Password" className="desktopHeader" redirectQuery={linkQuery} />
-        <Header origin="Reset Password" className="mobileHeader" redirectLabel="Signup" redirectPage="signup" redirectQuery={linkQuery} mobile />
 
+    if (this.state.resetSuccess) {
+      content = (
+        <div className="centerDiv smallWrapper">
+          <PasswordResetSuccess android="true"/>
```

Comment:
why is this "true" rather than conditional on some UA or URL query param? Also, as this is JSX this assigns the *string value* "true" to the property `android`. Using `android={true}` makes sure the boolean value is bound instead.